ansible/roles/hoover/templates/snoop2-localsettings.py

```diff
@@ -0,0 +1,15 @@
+from .defaultsettings import *
+from .secret_key import SECRET_KEY
+
+DEBUG = {% if devel %}True{% else %}False{% endif %}
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'hoover-snoop2',
+    }
+}
+
+ALLOWED_HOSTS = ["localhost"]
+
+SNOOP_TIKA_URL = 'http://localhost:15423'
```

Comment:
The [hoover/setup](https://github.com/hoover/setup/blob/snoop2/hoover_script.py#L387) writes `SNOOP_BLOB_STORAGE` but this template doesn't.